extensions/amp-story/1.0/events.js

```diff
@@ -57,30 +57,14 @@ export const EventType = {
   PAGE_LOADED: 'ampstory:pageload',
 };
 
-
-/**
- * @param {!EventTarget} source
- * @param {string} eventName
- * @param {boolean=} opt_bubbles
- */
-export function dispatch(source, eventName, opt_bubbles) {
-  const event = new Event(eventName, {bubbles: !!opt_bubbles});
-  if (event.initEvent) {
-    event.initEvent(eventName, /* bubbles */ !!opt_bubbles,
-        /* cancelable */ false);
-  }
-  source.dispatchEvent(event);
-}
-
-
 /**
  * @param {!Window} win
  * @param {!EventTarget} source
  * @param {string} eventName
- * @param {!Object} payload
+ * @param {!Object=} opt_payload
  * @param {!CustomEventInit=} opt_eventInit
  */
-export function dispatchCustom(win, source, eventName, payload, opt_eventInit) {
-  const event = createCustomEvent(win, eventName, payload, opt_eventInit);
+export function dispatch(win, source, eventName, opt_payload, opt_eventInit) {
```

Comment:
Nit: With the new JavaScript styleguide, you can use ES6 default parameters.

````
// This
dispatch(win, source, eventName, opt_payload, opt_eventInit) {
// Can become
dispatch(win, source, eventName, payload = undefined, eventInit = undefined) {
````

(See: https://google.github.io/styleguide/jsguide.html#features-functions-parameters)

Reply:
That makes sense! Should I also change the javadoc, so the payload and eventInit parameters look like the others instead of having an "=" in their type declarations?

Reply:
The `=` in the type declaration means that the value can be undefined (or not provided), so you want to keep it.
Changing `opt_payload` into `payload` in the parameters list and in the doc should do it :)